editor/scaffolds/playground-canvas/library.tsx

```
@@ -0,0 +1,257 @@
+              );
+            }}
+            items={icons}
+          />
+        </TabsContent>
+      </Tabs>
+    </>
+  );
+}
+
+type ReflectUIIconData = {
+  host: "material" | "ant-design" | "radix-ui" | "unicons" | (string | {});
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)